MSPageViewController/Source/MSPageViewController.m

```diff
@@ -31,6 +31,36 @@ - (id)initWithCoder:(NSCoder *)aDecoder {
     return self;
 }
 
+#pragma mark - Public
+
+- (NSInteger)pageCount {
```

Comment:
Why use an `NSInteger` and cast. Wouldn't just keeping it `NSUInteger` to match the collection `count` type be easier?